tests/Unit/NumericalAlgorithms/DiscontinuousGalerkin/Actions/Test_FluxCommunication.cpp

```diff
@@ -434,51 +429,31 @@ SPECTRE_TEST_CASE("Unit.DiscontinuousGalerkin.Actions.FluxCommunication",
   }
 
   // Now check ReceiveDataForFluxes
-  CHECK_FALSE(runner.is_ready<my_component,
-                              dg::Actions::ReceiveDataForFluxes<metavariables>>(
-      sent_box, self_id));
-  runner.apply<my_component, send_data_for_fluxes<metavariables>>(
-      runner.algorithms<my_component>()
-          .at(south_id)
-          .get_databox<initial_databox_type>(),
-      south_id);
-  CHECK_FALSE(runner.is_ready<my_component,
-                              dg::Actions::ReceiveDataForFluxes<metavariables>>(
-      sent_box, self_id));
-  runner.apply<my_component, send_data_for_fluxes<metavariables>>(
-      runner.algorithms<my_component>()
-          .at(east_id)
-          .get_databox<initial_databox_type>(),
-      south_id);
-  CHECK_FALSE(runner.is_ready<my_component,
-                              dg::Actions::ReceiveDataForFluxes<metavariables>>(
-      sent_box, self_id));
-  runner.apply<my_component, send_data_for_fluxes<metavariables>>(
-      runner.algorithms<my_component>()
-          .at(west_id)
-          .get_databox<initial_databox_type>(),
-      south_id);
-  CHECK(runner.is_ready<my_component,
-                        dg::Actions::ReceiveDataForFluxes<metavariables>>(
-      sent_box, self_id));
-
-  auto received_box = std::get<0>(
-      runner.apply<my_component,
-                   dg::Actions::ReceiveDataForFluxes<metavariables>>(sent_box,
-                                                                     self_id));
+  CHECK_FALSE(runner.is_ready<my_component>(self_id));
+  runner.next_action<my_component>(south_id);
+  CHECK_FALSE(runner.is_ready<my_component>(self_id));
+  runner.next_action<my_component>(east_id);
+  CHECK_FALSE(runner.is_ready<my_component>(self_id));
+  runner.next_action<my_component>(west_id);
+  CHECK(runner.is_ready<my_component>(self_id));
+
+  // DeceiveDataForFluxes
```

Comment:
I know we're mocking, but...

Reply:
I really enjoyed this typo, thank you for noticing it :)